ballerina/build.gradle

```diff
@@ -24,6 +24,7 @@ description = 'Ballerina - SAP S/4HANA Delivery and Transporation APIs'
 subprojects {
     ext {
         packageOrg = "ballerinax"
+        ballerinaTomlFilePlaceHolder = new File("${project.rootDir}/build-config/resources/Ballerina.toml")
```

Comment:
Remove this

Reply:
[Add PR review changes to gradle build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.delivery/pull/7/commits/031c361cb9b165454c9d51d179f5c52002682d85)